sap-spring-properties/src/main/java/com/sap/cloud/environment/servicebinding/SapSpringPropertiesServiceBindingAccessor.java

```diff
@@ -0,0 +1,104 @@
+package com.sap.cloud.environment.servicebinding;
+
+import com.sap.cloud.environment.servicebinding.api.DefaultServiceBinding;
+import com.sap.cloud.environment.servicebinding.api.ServiceBinding;
+import com.sap.cloud.environment.servicebinding.api.ServiceBindingAccessor;
+import com.sap.cloud.environment.servicebinding.api.exception.ServiceBindingAccessException;
+import com.sap.cloud.environment.servicebinding.environment.SapServiceBindingsPropertiesAccessor;
+import com.sap.cloud.environment.servicebinding.SapServiceBindingsProperties.ServiceBindingProperties;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+import java.util.stream.Collectors;
+
+/**
+ * A {@link ServiceBindingAccessor} that is able to load {@link SapServiceBindingsProperties}s from Spring's application
+ * properties.
+ */
+public class SapSpringPropertiesServiceBindingAccessor implements ServiceBindingAccessor {
+    @Nonnull
+    private static final Logger logger = LoggerFactory.getLogger(SapSpringPropertiesServiceBindingAccessor.class);
+
+    @Nonnull
+    @Override
+    public List<ServiceBinding> getServiceBindings()
+            throws ServiceBindingAccessException {
+        if (SapServiceBindingsPropertiesAccessor.getServiceBindingsProperties().isEmpty()) {
+            logger.info("No service bindings found in properties.");
+            return Collections.emptyList();
+        }
+
+        List<ServiceBinding> serviceBindings = fetchServiceBindings();
+
+        serviceBindings.forEach(binding ->
+                logger.info("Service binding {} found in properties.", binding.getServiceName().get()) //NOSONAR
+        );
+        return serviceBindings;
+    }
+
+    @Nonnull
+    private List<ServiceBinding> fetchServiceBindings() {
+        Map<String, ServiceBindingProperties> serviceBindingsProperties =
+                SapServiceBindingsPropertiesAccessor.getServiceBindingsProperties();
+        return serviceBindingsProperties
+                .entrySet()
+                .stream()
+                .map(entry -> toServiceBinding(entry.getKey(), entry.getValue()))
+                .filter(Objects::nonNull)
+                .collect(Collectors.toList());
+    }
+
+    @Nullable
+    private ServiceBinding toServiceBinding(
+            @Nonnull String serviceName,
+            @Nonnull final ServiceBindingProperties serviceBindingProperties) {
+        if (validateServiceBindingProperties(serviceBindingProperties)) {
+            return DefaultServiceBinding
+                    .builder()
+                    .copy(Collections.emptyMap())
+                    .withName(serviceBindingProperties.getName())
+                    .withServiceName(serviceName)
```

Comment:
I think the name should rather be the key instead of the serviceName in the properties map. There might be multiple bindings of the same service with different names. This is currently not possible.

Reply:
Sounds reasonable. I took this over from `SapVcapServicesServiceBindingAccessor`. I guess there it is wrong as well then:
```
[...]
 .withNameKey("name")
 .withServiceName(serviceName)
[...]
```
`serviceName` is the JSON key = binding name from VCAP

Reply:
In VCAP_SERVICES the JSON key is the service name (not the service instance or binding name). You also don't find a single object under that key, but an array containing multiple service instances/bindings.